src/ui/main/icons.ts

```
@@ -16,13 +16,28 @@ export const getAppIconPath = ({
+  badge: Server['badge'],
+  isLoggedIn: boolean
+): string => {
+  if (!isLoggedIn) {
+    return path.join(app.getAppPath(), 'app/images/tray/darwin/offline.png');
+  }
+
+  return path.join(
     app.getAppPath(),
     `app/images/tray/darwin/${badge ? 'notification' : 'default'}Template.png`
   );
+}
```

ESLint

[error] Insert `;`

(prettier/prettier)